microsoft/testsuites/network/common.py

```diff
@@ -36,12 +36,19 @@ def initialize_nic_info(
     for node in environment.nodes.list():
         if is_sriov:
             network_interface_feature = node.features[NetworkInterface]
+            # verify SRIOV in environment is enabled, wait for enable if not.
+            if not network_interface_feature.is_enabled_sriov():
+                network_interface_feature.switch_sriov(
+                    enable=True, wait=True, reset_connections=True
+                )
+
```

Comment:
Do you have an example of which fails for this reason? I think we should work on the case which will change the AN for the VM.

Reply:
Let me see if I can find it, I noticed some SRIOV tests fast-failing because SRIOV was not detected immediately upon test entry. Since we're reusing environments it occurred to me that we should wait for the VF for some amount of time before starting the test, it may be that it was just enabled and we need to wait a minute or two for the VF to show up.

Reply:
I don't think this is actually needed. Going to close